src/custom/utils/metadata.ts

```diff
@@ -1,3 +1,8 @@
+import IPFS from 'ipfs-core'
+import CID from 'cids'
+import multihashes from 'multihashes'
+import pinataSDK from '@pinata/sdk'
```

Comment:
We have a dir `custom/api` with API integrations. Maybe this fits more there? At least most of `uploadMetadataDocToIpfs` code

Reply:
FYI, the IPFS related changes are from https://github.com/gnosis/cowswap/pull/1393

I recommend you comment there :)

Reply:
Done :)